codalab/apps/web/static/js/vendor/jquery-editable-poshytip.js

```diff
@@ -2187,7 +2186,7 @@ Makes editable any HTML element on the page. Applied as jQuery method.
         @type string
         @default 'Empty'
         **/         
-        emptytext: 'Empty',
+        emptytext: '{empty}',
```

Comment:
oh. I did not know about this. Thanks for letting me know. :+1:  
Also, do we still want the none to be in italics?